tools/mo/openvino/tools/mo/convert.py

```diff
@@ -0,0 +1,1205 @@
+                        protobuf format. If not specified, the Model Optimizer
+                        treats it as a binary file by default.
+    :param input_checkpoint: TensorFlow*: variables file to load.
+    :param input_meta_graph: Tensorflow*: a file with a meta-graph of the model
+                        before freezing
+    :param saved_model_dir: TensorFlow*: directory with a model in SavedModel
+                        format of TensorFlow 1.x or 2.x version.
+    :param saved_model_tags: Group of tag(s) of the MetaGraphDef to load, in string
+                        format, separated by ','. For tag-set contains
+                        multiple tags, all tags must be passed in.
+    :param tensorflow_custom_operations_config_update: TensorFlow*: update the configuration file with node
+                        name patterns with input/output nodes information.
+    :param tensorflow_object_detection_api_pipeline_config: TensorFlow*: path to the pipeline configuration file
+                        used to generate model created with help of Object
+                        Detection API.
+    :param tensorboard_logdir: TensorFlow*: dump the input graph to a given directory
+                        that should be used with TensorBoard.
+    :param tensorflow_custom_layer_libraries: TensorFlow*: comma separated list of shared libraries
+                        with TensorFlow* custom operations implementation.
+    :param input_proto: Deploy-ready prototxt file that contains a topology
+                        structure and layer attributes
+    :param caffe_parser_path: Path to Python Caffe* parser generated from
+                        caffe.proto
+    :param k: Path to CustomLayersMapping.xml to register custom
+                        layers
+    :param input_symbol: Symbol file (for example, model-symbol.json) that
+                        contains a topology structure and layer attributes
+    :param nd_prefix_name: Prefix name for args.nd and argx.nd files.
+    :param pretrained_model_name: Name of a pretrained MXNet model without extension and
+                        epoch number. This model will be merged with args.nd
+                        and argx.nd files
+    :param save_params_from_nd: Enable saving built parameters file from .nd files
+    :param legacy_mxnet_model: Enable MXNet loader to make a model compatible with
+                        the latest MXNet version. Use only if your model was
+                        trained with MXNet version lower than 1.0.0
+    :param enable_ssd_gluoncv: Enable pattern matchers replacers for converting
+                        gluoncv ssd topologies.
+    :param counts: Path to the counts file
+    :param remove_output_softmax: Removes the SoftMax layer that is the output layer
+    :param remove_memory: Removes the Memory layer and use additional inputs
+                        outputs instead
+    """
```

Comment:
It looks to be a copied-and-pasted content from command line parameters definition. It is harder to maintain it in two places. Please think about having it once and reuse in both command line options and doc string.

BTW, the content for the two places is almost identical but not absolutely identical, because in Python API we allow passing python objects of specific types, this functionality is not available in cmdline. So we have to organize description of parameters in a smarter way to avoid duplicates and still having a slightly different content.

Reply:
I made it so that convert() shows all available params if used without arguments. Comments depend on whether used by mo.convert or cmdline. Please, take a look.